contributing.md

```diff
@@ -0,0 +1,164 @@
+
+A core developer reviews your request and either comments or approves the request. A core developer merges the approved request to the **main** branch usually when a newer version of the product is released.
+
+!!! note
+
+    We appreciate your work but the pull request may be redone to meet internal requirements.
+
+### Edit documentation online with GitHub
+
+In the upper right corner, select the pencil icon to open the source file in the GitHub editor. In this editor, you can make the changes, view the changes in the Preview tab, and create a pull request. The source `.md` file of the page opens in a GitHub editor in your browser. If you haven't worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).
+
+Edit the page using the Markdown syntax.
+
+You should review your changes on the **Preview** tab.
+
+Commit your changes to a pull request.
+
+In the **Commit changes** section, add a summary (72 characters or less) of what was changed
+  
+Then select the **Create a new branch for this commit and start a pull request** option. Accept the name for the branch and then Select **Commit changes**.
+
+GitHub creates a branch and a commit for your changes. The operation loads a page that shows the following:
+
+-  a base branch where you offer your changes
+  
+- your commit message
+  
+- a visual representation of your changes against the original page 
+
+You can make a review the page and click **Create pull request**.
+
+For more information, see [Editing files in GitHub](https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files) 
+
+### Edit documentation locally
+
+This option is for users are comfortable with [git] ([https://](https://git-scm.com/)) commands. 
+
+The steps are the following:
+
+1. Fork this repository
+
+2. Clone the forked repository to your machine:
+
+    ```shell
+        git clone git@github.com:<your_github_name>/proxysql-admin-tool-doc.git
+        cd <directory name>/proxysql-admin-tool-doc
+    ```
+
+3. Add the remote origin repository:
+
+    ```shell
+        git remote add origin https://github.com/percona/proxysql-admin-tool-doc.git
+    ```
+
+4. Checkout the appropriate branch and pull the latest changes from origin
+
+    ```shell
+        git checkout main && git pull origin main
+    ```
+
+5. Create a separate branch for your changes
+
+    ```shell
+        git checkout -b <my_changes>
+    ```
+
+6. The structure of the repository is:
+
+    <img src= "./docs/_images/treeview.png" alt="tree view of directory" width = 20% height = 20%)/>
+
+7. Work in the `/docs` directory. Add code examples, if necessary. We recommend that you check your changes using either a Preview built into your editor (if you have one) or build HTML to check it there. You can build HTML [on your machine](#Installing-Sphinx-and-building-locally) or [using Docker](#Using-Docker).
```

Comment:
on your machine](#Installing-Sphinx-and-building-locally) -> we do not use Sphinx any more

Reply:
Fixed.